apps/voting/contracts/Voting.sol

```diff
@@ -363,36 +476,42 @@ contract Voting is IForwarder, AragonApp {
         emit StartVote(voteId, msg.sender, _metadata);
 
         if (_castVote && _canVote(voteId, msg.sender)) {
-            _vote(voteId, true, msg.sender);
+            _vote(voteId, true, msg.sender, false);
         }
     }
 
     /**
     * @dev Internal function to cast a vote or object to.
       @dev It assumes that voter can support or object to the vote
     */
-    function _vote(uint256 _voteId, bool _supports, address _voter) internal {
+    function _vote(uint256 _voteId, bool _supports, address _voter, bool _isManager) internal {
         Vote storage vote_ = votes[_voteId];
 
         // This could re-enter, though we can assume the governance token is not malicious
         uint256 voterStake = token.balanceOfAt(_voter, vote_.snapshotBlock);
         VoterState state = vote_.voters[_voter];
 
+        // Voting manager can't overwrite holder vote
+        if (_isManager && (state == VoterState.Yea || state == VoterState.Nay)) {
+            revert(ERROR_MANAGER_CANNOT_OVERWRITE_VOTE);
```

Comment:
in this design, if the delegate votes for an array of addresses (calls _vote from voteForMultiple) and one of the token holders voted himself, then the delegate will no longer be able to vote for the remaining ones as the whole tx will revert

Reply:
Or have a UI method filtering "of addresses delegated to me who voted already"; needs to have a comment for auditors also

Reply:
Alternatively 1) move "revert" up the _vote method 2) account if singular voteFor passed or not in cycle of voteForMultiple 3) at the end of voteForMultiple get event with vpVoted addressesVotedFor vpSkipped addressesSkipped